docs/samples/deployments/app-service/presidio-services.json

```diff
@@ -135,12 +142,69 @@
+        "imageRedactorServicePlanName": "[if(parameters('singleAppServicePlan'), variables('sharedAppServicePlanName'), concat(parameters('imageRedactorWebAppName'), '-ServicePlan'))]",
+        "analyzerServicePlanSku": "[if(parameters('singleAppServicePlan'), variables('sharedAppServicePlanSku'), parameters('analyzerSku'))]",
+        "anonymizerServicePlanSku": "[if(parameters('singleAppServicePlan'), variables('sharedAppServicePlanSku'), parameters('anonymizerSku'))]",
+        "imageRedactorServicePlanSku": "[if(parameters('singleAppServicePlan'), variables('sharedAppServicePlanSku'), parameters('imageRedactorSku'))]"
+    },
     "resources": [
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2019-10-01",
+            "name": "presidio-anonymizer",
+            "condition": "[parameters('anonymizer')]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/microsoft/presidio/main/docs/samples/deployments/app-service/presidio-app-service.json",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "webAppName": {
+                        "value": "[parameters('anonymizerWebAppName')]"
+                    },
+                    "appServicePlanName": {
+                        "value": "[variables('anonymizerServicePlanName')]"
+                    },
+                    "sku": {
+                        "value": "[variables('anonymizerServicePlanSku')]"
+                    },
+                    "location": {
+                        "value": "[parameters('location')]"
+                    },
+                    "imageIsPublic": {
+                        "value": "[parameters('imagesArePublic')]"
+                    },
+                    "image": {
+                        "value": "[parameters('anonymizerImage')]"
+                    },
+                    "dockerRegsitryUrl": {
+                        "value": "[parameters('dockerRegsitryUrl')]"
+                    },
+                    "dockerRegistryUser": {
+                        "value": "[parameters('dockerRegistryUser')]"
+                    },
+                    "dockerRegistryPassword": {
+                        "value": "[parameters('dockerRegistryPassword')]"
+                    },
+                    "ipRestriction": {
+                        "value": "[parameters('ipRestriction')]"
+                    }
+                }
+            }
+        },
         {
             "type": "Microsoft.Resources/deployments",
             "apiVersion": "2019-10-01",
             "name": "presidio-analyzer",
+            "dependsOn": [
+                "[resourceId('Microsoft.Resources/deployments', 'presidio-anonymizer')]"
```

Comment:
what happens if the anonymizer will not be deployed?

Reply:
nice catch!

Reply:
didn't need the dependsOn. ARM knows how to handle it.